docusaurus.config.js

```diff
@@ -549,10 +552,10 @@ module.exports = {
             "/docs/**/assets/**",
             "/docs/**/tags/**",
             "/docs/next/**",
-            "/docs/8.7/**",
             "/docs/8.3/**",
```

Comment:
The 8.3 banner was removed above, should 8.3 be removed from this section too?

Reply:
No, this is to keep 8.3 out of the sitemap.